tests/results/SourceRangeTest.py

```diff
@@ -86,6 +86,18 @@ def test_json(self):
                                       end_line=4).__json__(use_relpath=True)
         self.assertEqual(uut['start'], self.result_fileB_line2)
 
+    def test_str(self):
+        uut = str(SourceRange.from_values('test_file', 1, 1, 2, 1))
+        compare = abspath('test_file') + ': L1 C1 : L2 C1'
+        self.assertEqual(uut, compare)
+
+        uut = str(SourceRange.from_values('test_file', 1, None, 2, None))
+        compare = abspath('test_file') + ': L1 : L2'
+        self.assertEqual(uut, compare)
+
+        uut = str(SourceRange.from_values('test_file', None, None, None, None))
+        self.assertEqual(uut, abspath('test_file'))
```

Comment:
Technically, we get numbers right - It says `start.line` is interpreted as the first line when it is `None`, but it will be just be 1 and not `None`. And, each `Bear` ( whichever other API or executables we call to get the bear work ) will give **both** column numbers or it just wont give both. And, the same goes with the line.
I think we should stick to these 3 boundary cases. It is very unlikely that the above discussed cases will occur. What do you think?